### PR TITLE
Add ability to render markdown; includes copyright page as first example

### PR DESCRIPTION
**Pull request by saragibby**
The way the dynamic routing is working here:
1. If routing to a page that exists that will take precedence (i.e. https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/partners)
2. If not a found page it will fall into the `[markdown]` page:
  A. If markdown file for that path exists it will render that markdown (i.e. https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/copyright)
  B. If no markdown file for that path exists it will render the error page not found page as it currently does (i.e. https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/bacon)

For a first pass I am pulling event values from graph to use in the markdown. I currently have them flattening to keys such as `event_name`, `event_slogan`, etc. The replace will find any of those text string in the markdown and replace them with the value returned from the query.

### CODE REVIEW

**Comment by csell5** [comment]
#### 📝Changed routes:


| [**/wi/\[markdown\]**](https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/[markdown]) |
|:-:|
|<a href="https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/[markdown]">Screenshot of /wi/[markdown]</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/xSpGApu5WGvm4uyCgYzfbeuZ6jE_">(view full size)</a> |







Commit 61404fc4eff4a5d213965792c38d3861d5be470d (https://that-website-git-kb-pocmarkdown.that-conference.now.sh).

**Comment by GregLevenhagen** [comment]
I have things working with variables coming from an API call and the dynamic routing. Browse to https://that-website-5eguhch75.now.sh/wi/bacon to check out.

**Comment by kburnell** [comment]
This looks very promising @GregLevenhagen.  Nice job!

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
     "@apollo/react-ssr": "^3.1.3",
+    "@mdx-js/loader": "^1.5.1",
+    "@next/mdx": "^9.1.4",
```
Comment (GregLevenhagen) [inline package.json]: I don't believe this branch is using anything from MDX. Should mdx-js/loader and next/mdx be included here?

**Comment by saragibby** [comment]
Agreed. Nicely done @GregLevenhagen! I'm going to pull down the changes and do some clean up to tag you all back in so we can get these merged in to start working with.

**Comment by saragibby** [comment]
The way the dynamic routing is working here:
1. If routing to a page that exists that will take precedence (i.e. https://that-website-git-kb-pocmarkdown.that-conference.now.sh/wi/partners)
2. If not a found page it will fall into the `[markdown]` page:
  A. If markdown file for that path exists it will render that markdown.
  B. If no markdown file for that path exists it will render the error page not found page as it currently does

**Comment by GregLevenhagen** [comment]
Nice job @saragibby on getting it ready to go-live! Agreed, great team effort here!

**Pull request merged**